zilliqa/src/node.rs

```diff
@@ -433,20 +433,50 @@ impl Node {
             return Ok(());
         };
 
+        // Need to get and send corresponding TXs for the block
+        let txs: Vec<SignedTransaction> = block
```

Comment:
Is there a reason not to use the new `block_to_proposal()` function added just below? It seems to duplicate this logic exactly (unless I've missed some detail!)

Reply:
ah, that is an oversight! thanks.